task-sdk/src/airflow/sdk/__init__.py

```diff
@@ -120,7 +123,7 @@ def __getattr__(name: str):
         import importlib
 
         mod = importlib.import_module(module_path, __name__)
-        val = getattr(mod, name)
+        val = getattr(mod, name, mod)
```

Comment:
Needed for cases like "macros". If the "name" is not found in getattr, we should return the module name that is resolved -- like for macros, `airflow.sdk.definitions.macros` is resolved